response_operations_ui/controllers/message_controllers.py

```diff
@@ -33,3 +33,20 @@ def get_message_list(params):
     except KeyError:
         logger.exception("Response was successful but didn't contain a 'messages' key")
         raise NoMessagesError
+
+
+def send_message(message_json):
+    logger.debug("Sending message")
+
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/send-message'
+    # This will be removed once UAA is completed.  For now we need the call to backstage to include
```

Comment:
Would it be better putting this as a todo rather than a comment?

Reply:
We have tasks on the board to make these changes so I would argue a comment is sufficient.